pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1346,7 +1047,428 @@ func (r *ReconcileInfinispan) serviceExternal(m *infinispanv1.Infinispan) *corev
 	if m.Spec.Expose.NodePort > 0 {
 		externalService.Spec.Ports[0].NodePort = m.Spec.Expose.NodePort
 	}
-	// Set Infinispan instance as the owner and controller
-	controllerutil.SetControllerReference(m, externalService, r.scheme)
 	return externalService
 }
+
+// reconcileXSite creates the xsite service if needed
+func (r *ReconcileInfinispan) reconcileXSite(ispn *infinispanv1.Infinispan, scheme *runtime.Scheme, logger logr.Logger) (*corev1.Service, error) {
+	siteServiceName := ispn.GetSiteServiceName()
+	siteService, err := configuration.GetOrCreateSiteService(siteServiceName, ispn, kubernetes.Client, scheme, logger)
+	if err != nil {
+		logger.Error(err, "could not get or create site service")
+		return nil, err
+	}
+	return siteService, nil
+}
+
+// reconcileConfigMap creates the configMap for Infinispan if needed
+func (r *ReconcileInfinispan) reconcileConfigMap(ispn *infinispanv1.Infinispan, configMap *corev1.ConfigMap, logger logr.Logger) error {
+	cm := &corev1.ConfigMap{}
+	ns := types.NamespacedName{
+		Name:      configMap.Name,
+		Namespace: configMap.Namespace,
+	}
+	err := r.client.Get(context.TODO(), ns, cm)
```

Comment:
```suggestion
	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: configMap.Namespace, Name: configMap.Name}, cm)
```
@rigazilla maybe we can change all places where `ns` are using for compacting code?

Reply:
> @rigazilla maybe we can change all places where `ns` are using for compacting code?

sounds good, I've changed it all around.